recipes/isitgr/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Where is this file from? It's not in the upstream repository

Reply:
Oh, I got it from https://choosealicense.com/licenses/mit/, should I change it?

Reply:
Ah you're the author of isitgr right? If so, it's fine provided Mustapha Ishak is also happy to use the MIT licence.

It's important that the copyright owners of the code choose the licence as they're the only ones who can grant permissions to other people.

Normally people put the licence in the top directory of the git repository so the licence is easy to find and kept with the code. At the moment if I clone the GitHub repo it's a violation of this line of the licence:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

Reply:
I am happy to use  the MIT licence.